gatsby-node.js

```diff
@@ -76,6 +76,29 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
   createRedirect({ fromPath: "/cloud-native-management/gitops", toPath: "/solutions/gitops", redirectInBrowser: true, isPermanent: true });
   createRedirect({ fromPath: "/cloud-native-management/gitops/snapshot", toPath: "/solutions/gitops/snapshot", redirectInBrowser: true, isPermanent: true });
   createRedirect({ fromPath: "/cloud-native-management/gitops/performance-management", toPath: "/solutions/gitops/performance-management", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/cloud-native-management/meshery/integrations/aws-node-termination-handler-2", toPath: "/cloud-native-management/meshery/integrations/aws-node-termination-handler", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/cloud-native-management/meshery/integrations/aws-node-termination-handler-2/", toPath: "/cloud-native-management/meshery/integrations/aws-node-termination-handler", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/cloud-native-management/meshery/integrations/service-mesh-interface-smi.html", toPath: "/projects/service-mesh-interface-conformance", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/cloud-native-management/gcp-diagram", toPath: "/cloud-native-management/generate-gcp-architecture-diagram", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/whiteboard", toPath: "/solutions/architecture-diagram", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/comments", toPath: "/cloud-native-management/meshmap/collaborate/peer-reviews", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/careers/company/legal/terms-of-service", toPath: "/company/legal/terms-of-service", redirectInBrowser: true, isPermanent: true });
```

Comment:
Where is this invalid link coming from? Can we correct this at source instead of redirecting?

Reply:
@leecalcote, I searched for each of these links in our codebase, but none of them exist within our codebase. Therefore, these are external links from other websites